app/views/layouts/application.html.erb

```diff
@@ -7,6 +7,7 @@
     <%= stylesheet_link_tag    "application" %>
     <%= javascript_include_tag "vendor/modernizr" %>
     <%= javascript_include_tag "application", 'data-turbolinks-track' => true %>
+    <%= javascript_include_tag "https://cdnjs.cloudflare.com/ajax/libs/angular-filter/0.5.8/angular-filter.min.js" %>
```

Comment:
Maybe pull in this file into Vendor folder and reference it in the application.js file